run-profile

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+# Assume that assignments have been generated into ../zeth/build
+
+ASSIGNMENTS_DIR=../zeth/build
+ZETH_TOOL=../zeth/build/zeth_tool/zeth-tool
+
+# 1 - jsin/jsout value
+function setup_files() {
+    # Unpack data from the archive, either from
+    # ${STORE_HOST}:${STORE_PATH}, or the local ${ASSIGNMENTS_DIR}
+    if ! [ -e assignment_$1_$1.bin ] ; then
```

Comment:
So we assume JSIN = JSOUT here, right?

Reply:
Yes.  Exactly.  I've used the expression JSIN/JSOUT in comments to mean this when referring to a single value (e.g. in comments above the functions).  Hopefully it's clear, but otherwise we can reword.

Reply:
It is clear, thanks